### PR TITLE
Allow configure label_catalogue, label_translator_strategy, group and…

### LINKED ISSUE #7: Setting defaults using the configuration
How about setting defaults using the configuration? Sometimes I need annotations just to change the `labelTranslatorStrategy `or the `labelCatalogue`.

```yml
sonata_auto_configure:
    defaults:
        label_catalogue: admin
        label_translator_strategy: sonata.admin.label.strategy.underscore
        show_mosaic_button: false
```

### PR DESCRIPTION
**Pull request by gremo**
Closes #7 

```markdown
### Added
- bundle configuration for additional info for sonata admin tag

### CODE REVIEW

**Comment by gremo** [comment]
There is a design problem with boolean values in annotation, we should work on this before adding other options to the configuration (like keep_open, on_top, etc).

**Comment by kunicmarko20** [comment]
can you explain the problem?

File: src/DependencyInjection/Compiler/AutoConfigureAdminClassesCompilerPass.php [hunk 1]
```diff
@@ -74,7 +74,14 @@ public function process(ContainerBuilder $container): void
                 AdminOptions::class
             ) ?? new AdminOptions();
 
-            $this->setDefaultValuesForAnnotation($annotation, $name);
+            $annotationDefaults['label_catalogue'] = $container
```
Comment (kunicmarko20) [inline src/DependencyInjection/Compiler/AutoConfigureAdminClassesCompilerPass.php]: can you extract `$annotationDefaults` outside of `foreach`

**Review by kunicmarko20 — COMMENTED** [review]
looks fine, can you please update the tests.

**Comment by gremo** [comment]
> can you explain the problem?

You are providing defaults in `AdminOptions`, which is wrong IMHO. For example, `showInDashboard `is `true `by default, as `keepOpen` is false by `default`.

Here is the problem: say that I add another option, for example `keep_open` in configuration:

```yml
sonata_auto_configure:
    admin:
        keep_open: true
        # ...
```
This "global" option will be used only if you set `keepOpen=null` in every admin. Because this check:

```php
if (null === $annotation->keepOpen) {
    $annotation->keepOpen= $defaults['keep_open'];
}
```

**Comment by kunicmarko20** [comment]
If you add the same values as we currently have in the annotation to the config, and drop the annotation default values. It would be BC and no one would see the difference, right?

**Comment by gremo** [comment]
> If you add the same values as we currently have in the annotation to the config, and drop the annotation default values. It would be BC and no one would see the difference, right?

Yes, that would be a BC. 

But you should really consider to not provide defaults, and let SonataAdminBundle process all admin and setting defaults by itself. That is, only change `showInDashboard`, `keepOpen` and `onTop`... maybe adding them to the configuration.

**Comment by kunicmarko20** [comment]
Hm, if admin sets the same values as the default ones, I guess removing them isn't a bc break at all, do we even need them as default then?

**Comment by gremo** [comment]
> Hm, if admin sets the same values as the default ones, I guess removing them isn't a bc break at all, do we even need them as default then?

Well, I can't explain better because my english is not very well 🤣 . I'll open a dedicated issue maybe? 
In the meanwhile, are we ok with this PR?

**Comment by kunicmarko20** [comment]
Implementation is good, if you update the tests to check this I will merge it.

**Pull request merged**